pwiz_aux/msrc/utility/vendor_api/Waters/WatersRawFile.hpp

```diff
@@ -457,6 +457,21 @@ struct PWIZ_API_DECL RawData
         return success;
     }
 
+    bool GetIsolationWindow(float& lowerOffset, float& upperOffset) {
+        MassLynxParameters parameters = DDAProcessor.GetParameters();
+        float lowerOffsetParam = stof(parameters.Get(DDAParameter::LOWEROFFSET));
+        float upperOffsetParam = stof(parameters.Get(DDAParameter::UPPEROFFSET));

```

Comment:
Thanks for the PR!

Just a few things changes would be nice here:
- existing style is open brace on new line and one line if statements don't need braces
- use lexical_cast<float> instead of stof (it's actually overloaded to use strtod but we can consistently check for bad_lexical_cast)
- how big is this parameters object that's going to get copied for every scan? Are those parameters the same for every scan?

Reply:
Hi, these were my changes that Pete has created the PR for. On the first two points, yep all fair, I'll make those changes. On the last point we believe that current instruments it will be the same for each scan, but there's the possibility of upcoming instruments varying by scan. The parameters file only contains the two values seen here so we'd prefer to leave as is.